File: 3226-minimum-number-game/minimum-number-game.py
class Solution:
    def numberGame(self, nums: List[int]) -> List[int]:
        nums.sort()
        bob_ptr , alice_ptr = 1 , 0
        while(alice_ptr < len(nums) or bob_ptr< len(nums)):
            nums[alice_ptr] , nums[bob_ptr] = nums[bob_ptr] , nums[alice_ptr]
            alice_ptr +=2
            bob_ptr +=2
        return(nums)
                

File: 3226-minimum-number-game/minimum-number-game.cs
public class Solution {
    public int[] NumberGame(int[] nums) {
        Array.Sort(nums);
			int low  = 0 ;
			int high = 1;
			while (high < nums.Length)
			{
				int temp = nums[low];
				nums[low] = nums[high];
				nums[high] = temp;
				low+=2;
				high+=2;
			}
            return nums;



    }
}